app/src/tx_reviewer.rs

```diff
@@ -505,7 +505,7 @@ fn bytes_to_string(bytes: &[u8]) -> Result<&str, ErrorCode> {
 }
 
 fn review<'a>(fields: &'a [Field<'a>], review_message: &str) -> Result<(), ErrorCode> {
-    let review_messages = [review_message];
+    let review_messages = ["Review ", review_message];
```

Comment:
I have added the `Review` prefix back. Currently the display is similar to app-bitcoin, but we need to add a space after these messages due to this issue: https://github.com/LedgerHQ/ledger-device-rust-sdk/issues/158